main.nf

```diff
@@ -689,16 +707,23 @@ process stringtieFPKM {
     stdout into stringtie_log
 
     script:
+    
+    if (params.forward_stranded){
+        def StringTie_direction = "--fr"
```

Comment:
I think you need to define this outside of the if statement, otherwise it'll be undefined when not stranded.

```groovy
def StringTie_direction = ''
if(params.forward_stranded){
    StringTie_direction = "--fr"
    // ...
```